uswds/uswds
# USWDS - Icon: Resolve a11y errors

## Summary
Removed `aria-label` from the `.message-holder` span to resolve an Axe a11y error. 

## Related issue
Closes #5005 

## Preview link
Preview link: [Icon](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/al-icon-a11y-error/iframe.html?args=&id=design-tokens-icons--icons&viewMode=story)

## Problem statement
Adding an `aria-label` to a span with no explicit `role` causes the following Axe errors: 

![image](https://user-images.githubusercontent.com/93996430/198131417-0c512b08-77d9-4ab1-b7de-108ef7189169.png)

## Solution
The `aria-label` did not seem to be necessary on the `.message-holder` span. This PR removes the `aria-label`. 

## Testing and review
1. Confirm that the `.message-holder` span does not need an `aria-label`.
1. Open the [preview link](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/al-icon-a11y-error/iframe.html?args=&id=design-tokens-icons--icons&viewMode=story) and run axe devTools. Confirm that all relevant errors have been resolved. 

---

Before opening this PR, make sure you’ve done whichever of these applies to you:
